packages/shared/src/logger/default-logger.ts

```diff
@@ -3,6 +3,15 @@
 import type { Logger } from './logger';
 
 export class DefaultLogger implements Logger {
+  
+  private readonly showInfo : boolean = false;
+  private readonly showDebug : boolean = false;
+
+  constructor(showInfo: boolean = false, showDebug: boolean = false){
```

Comment:
https://github.com/open-feature/js-sdk/issues/998#issuecomment-2541526479

Please insider this @kevinmlong.
Having the level configurable via env vars is common and most useful when deployed. Having to rebuild your Dockerfile or even commit a changed log level in case you need to observe something in a deployed instance would not be too practical to me.

Reply:
@lukas-reining - I'm all for being able to pass in configuration from the environment. Perhaps I'm confused though. Wouldn't the applications making use of the hook and the DefaultLogger be where the parameters passed in come from the environment? Perhaps the switch to passing in a LogLevel parameter resolved this as the the client implementation of the SDK would pass in `process.env.LOG_LEVEL` where appropriate.